modules/network-observability-loki-install.adoc

```diff
@@ -34,7 +34,7 @@ There are several ways you can install Loki. One way you can install the Loki Op
 +
 . Create a `Secret` YAML file. You can create this secret in the web console or CLI. 
 .. Using the web console, navigate to the *Project* -> *All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
-.. Navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. It is important to create this YAML file in the `netobserv` namespace that uses the `access_key_id` and `access_key_secret` to specify your credentials. 
+.. Navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. 
```

Comment:
@jpinsonneau @nathan-weinberg I removed the "same namespace" language in the docs. There were two places, here in the installing loki and another in the creating a LokiStack. I also added a call out to the `spec.loki.tls.caCert.namespace` field for awareness that this is an option now. 

I know this was already possible as of 1.3, but in 1.3 we only added it as a bug fix release note and didn't actually update the documentation. Now, its encapsulated in this "Easier config" feature, so I think we can remove the mentions from the doc.

Reply:
sure, maybe just highlight that it's optionnal
https://github.com/openshift/openshift-docs/pull/63276/files#r1311475502